test/Incremental/gen-output-file-map.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
Would be great to use this everywhere a test needs an OFM!
Even cooler if it could be in Swift!

Reply:
For the two points:
1) I'd really prefer we kill off this script and resolve [this SR](https://bugs.swift.org/browse/SR-6079)
2) I'd love to have written this in Swift - my knowledge of Python is slim - but we'd probably introduce dependencies on a number of packages in order to run the tests which is less than ideal. The python code is also invariant with respect to any bugs we may have in any given revision of the compiler.